numpy/fft/__init__.py

```diff
@@ -200,12 +200,13 @@
 
 """
 
-from . import _pocketfft, helper
+from . import _pocketfft, _helper
+from . import helper  # TODO: to be deleted once people move to `np.fft`
```

Comment:
This comment should say it's deprecated in Numpy 2.0, following similar comments in the codebase so it's greppable.